### PR TITLE
Webpack

### PR DESCRIPTION
**Pull request by drg101**

### CODE REVIEW

**Comment by pierce-smith1** [comment]
image
Developing locally - I have a hunch this isn't supposed to happen

**Comment by pierce-smith1** [comment]
Tried to run an npm update, got this:
image

**Comment by drg101** [comment]
@pierce-smith1 Can you try removing your `node_module` dir and doing `npm i`

**Review by pierce-smith1 — CHANGES_REQUESTED** [review]
The site seems to throw 404 errors for "/[object%20Module]" when trying to render icons on the map, e.g. for electrical substations. Given the "Module" this seems to be a webpack issue not dealing with images properly - not sure how to resolve this issue yet.

**Review by pierce-smith1 — APPROVED** [review]
Unable to find any more problems.

File: README.md [hunk 1]
```diff
@@ -17,10 +17,22 @@ Aperture's client
...
-    /src
-        /iframe           # Where the frame code is, read this directories' README for a full description
-        /dependencies     # index.html's dependencies
+    /src          # where the source code is
+        /css               # css
+        /static            # code that isn't really a library, and depends on a bunch of external items existing to be run
+        /third-party       # where third-party css lives  
+        /js                # where javascript for the frames live
+            /control           # js that affects the control of the frames, for instance, onclick listeners
+            /frame-specific    # js that is specific to a certain frame
```
Comment (pierce-smith1) [inline README.md:28]: That being said, this folder doesn't exist anymore so it should probably be taken out of the readme

**Pull request merged**